composer.json

```diff
@@ -5,26 +5,31 @@
     "description": "Bundle that provides controllers and services for using Sulu as headless content management system",
     "require": {
         "php": "^7.2",
-        "sulu/sulu": "^2.0.1"
+        "sulu/sulu": "^2.0.1",
+        "symfony/config": "^4.3 || ^5.0",
+        "symfony/dependency-injection": "^4.3 || ^5.0",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/http-foundation": "^4.3 || ^5.0",
+        "symfony/http-kernel": "^4.3 || ^5.0"
     },
     "require-dev": {
-        "coduo/php-matcher": "^3.2",
-        "handcraftedinthealps/code-coverage-checker": "^0.2.0",
+        "coduo/php-matcher": "^3.2 || ^4.0",
```

Comment:
For me it looks like the new version 4.0 does support the same range of symfony or other packages. As `coduo/php-matcher` is a dev dependency I think we can skip here ^3.2 or did you run into any problems skipping 3.2?

Reply:
Can't install `symfony/framework-bundle:^5.0` with `codue/php-matcher:^3.2`

Reply:
Thats clear I mean why you did not use `"coduo/php-matcher": "^4.0",` here? Dev dependencies aslong as they support the same dependencies graph as before can be upgraded to newest version.

Reply:
fixed